feature-nft-impl/src/main/java/io/novafoundation/nova/feature_nft_impl/data/source/NftProvidersRegistry.kt

```diff
@@ -24,7 +24,7 @@ class NftProvidersRegistry(
         }
     }
 
-    fun isAvailableChain(chain: Chain): Boolean {
+    fun isNftTransferAvailableOnChain(chain: Chain): Boolean {
         return chain.id in setOf(Chain.Geneses.STATEMINE)
     }
```

Comment:
Given that we now have `NftTransfer.areTransfersSupported()` I suggest delegating implementation of this function to `NftTransfer`

Reply:
Done